.storybook/utils.tsx

```diff
@@ -112,3 +117,15 @@ export const globalDocsPage: typeof DocsPage = () => (
     <Description />
   </React.Fragment>
 );
+
+export const handleComponentAttributes = (
```

Comment:
Since this is only filtering, can you rename it to convey this? I'll revive this PR to introduce a helper with more options: https://github.com/Esri/calcite-components/pull/1424.